src/tagging/tests/__snapshots__/value.modifier.test.js.snap

```diff
@@ -28,12 +28,7 @@ exports[`TagCategory Component match snapshot 1`] = `
     <ValueSelector
       multiValue={false}
       onTagValueChange={[MockFunction]}
-      selectedOption={
```

Comment:
Do you know what triggered this snapshot change? I can't see this test object being removed or changed anywhere.

Reply:
I did not update the test and I missed it. Thanks.